src/com/facebook/buck/ide/intellij/lang/android/AndroidLibraryModuleRule.java

```diff
@@ -53,6 +53,7 @@ public void apply(
     Optional<Path> dummyRDotJavaClassPath = moduleFactoryResolver.getDummyRDotJavaPath(target);
     if (dummyRDotJavaClassPath.isPresent()) {
       context.addExtraClassPathDependency(dummyRDotJavaClassPath.get());
```

Comment:
Why this logic doesn't work for you? This should create a library with extra class path which should point to R.java.